scripts/update_robots.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2021 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -eu
+
+repo_root="$(dirname "$0")/.."
+robots_txt_path="${repo_root}/docs/_static/robots.txt"
+version_from_script=$("${repo_root}"/scripts/get_version.sh | perl -pe 'chomp')
```

Comment:
Please don't add perl as a dependency.

Reply:
I don't get any unnecessary whitespace from `get_version.sh` - maybe you can just use
```suggestion
version_from_script=$("${repo_root}"/scripts/get_version.sh)
```
here?

Reply:
Removed Perl and used `tr`. It's for removing the trailing new line from `get_version.sh`

Reply:
Are you sure it is needed? I think the shell already removes the trailing newline on its own...